public/v4/apps/searx.yml

```diff
@@ -0,0 +1,96 @@
+            containerHttpPort: '3000'
+    # Morty
+    $$cap_appname-morty:
+        restart: always
+        environment:
+            MORTY_KEY: $$cap_MORTY_KEY
+            MORTY_ADDRESS: 0.0.0.0:3000
+        caproverExtra:
+            containerHttpPort: '3000'
+            dockerfileLines:
+                - FROM dalf/morty@$$cap_morty_version
+                - CMD -listen 0.0.0.0:3000 -timeout 6 -ipv6
+    # Checker
+    $$cap_appname-checker:
+        restart: always
+        volumes:
+            - $$cap_appname-checker-data:/usr/local/searx-checker/html/data
+        caproverExtra:
+            notExposeAsWebApp: true
+            dockerfileLines:
+                - FROM searx/searx-checker@$$cap_checker_version
+                - CMD -o html/data/status.json http://srv-captain--$$cap_appname:8080
+caproverOneClickApp:
+    variables:
+        - id: $$cap_searx_version
+          label: Searx Version Tag
+          defaultValue: '0.17.0-281-219af243'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/searx/searx/tags/
+        - id: $$cap_filtron_version
+          label: Filtron Docker Version Hash
+          defaultValue: 'sha256:5b55f79b06fdd58122a6d764eca09d2ec687c5af5b78969124e69714d6bb889b'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/dalf/filtron/tags/
+        - id: $$cap_morty_version
+          label: Morty Docker Version Hash
+          defaultValue: 'sha256:cb930c25254337e7a3abaf6daab326c663a4cdf1cbf4ae040ff7f670493df7fd'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/dalf/morty/tags/
+        - id: $$cap_checker_version
+          label: Searx-checker Docker Version Hash
+          defaultValue: 'sha256:8ebda54ed8854e1300a4670e2bba86a5d2f4ceeb5c39290e4cc20df22bf974be'
+          description: Starts with 'sha256:'.  Find on https://hub.docker.com/r/searx/searx-checker/tags/
+        - label: Searx base url
+          description: Base url to your Searx
+          defaultValue: https://searx.app.gocloud.fun
+          id: $$cap_SEARX_BASEURL
+        - label: Morty URL
+          description: Base url to your Morty
+          defaultValue: https://searx-morty.app.gocloud.fun
+          id: $$cap_MORTY_URL
+        - label: Morty Secret Key
+          description: use openssl rand -base64 33
+          defaultValue: RemplaceWithARealKey!
+          id: $$cap_MORTY_KEY
+        - label: Path to Filtron rules.json file
+          defaultValue: /var/lib/docker/volumes/captain--searx-filtron-rules/_data/rules.json
+          id: $$cap_FILTRON_RULES
+    instructions:
+        start: >-
+            Privacy-respecting, hackable metasearch engine.
+
+            Mayday! Mayday! Mayday!
+
+            First of all put the content of https://github.com/searx/searx-docker/blob/master/rules.json
```

Comment:
If it's literally just this file, why not use this line in Dockerfile?
```
ADD https://github.com/searx/searx-docker/blob/master/rules.json /etc/filtron/rules.json
```

Then we don't even need to cross map the volumes

Reply:
Learned something new!
Thx